templates/blocks/p4_structured_data.twig

```diff
@@ -0,0 +1,31 @@
+{% set social_menu_links = [] %}
+{% if footer_social_menu %}
+	{% for menu_details in footer_social_menu %}
+		{% set social_menu_links = social_menu_links|merge([menu_details.url]) %}
+	{% endfor %}
+{% endif %}
+{% set website_info = {
+	'@context': 'http://schema.org',
+	'@type': 'WebSite',
+	'url': site.link,
+	'name': site.name,
+	'alternateName': 'Greenpeace',
+	'potentialAction': {
+		'@type': 'SearchAction',
+		'target': ( site.link ~ '/?s={search_term_string}&orderby=_score' ),
+		'query-input': 'required name=search_term_string',
```

Comment:
I think the {search_term_string} is a var defined in the USA website. We probably need to do
something similar like
```
{% set search_term_string = fn('get_search_query') %}
'target': ( site.link ~ '/?s=' ~ search_term_string ~ '&orderby=_score' ),
```
As for the `query-input` are you sure this is the correct value for it? Might need to check here https://schema.org/SearchAction